packages/miniflare/src/plugins/kv/assets.ts

```diff
@@ -20,15 +23,14 @@ export function isWorkersWithAssets(
 
 const SERVICE_NAMESPACE_ASSET = `${KV_PLUGIN_NAME}:asset`;
 
-export function buildAssetsManifest(): Uint8Array {
-	const buffer = new ArrayBuffer(20);
-	const assetManifest = new Uint8Array(buffer); // [0, 0, 0, ..., 0, 0]
-	// this will signal to Asset Server Worker that its running in a
-	// local dev "context"
-	assetManifest.set([1], 0); // [1, 0, 0, ..., 0, 0]
-
-	return assetManifest;
-}
+export const buildAssetsManifest = async (dir: string) => {
+	const assetManifest = await walk(dir);
+	const sortedAssetManifest = sortManifest(assetManifest);
+	const encodedAssetManifest = encodeManifest(sortedAssetManifest);
+	// to indicate it is in dev:
+	encodedAssetManifest.set([1], 0);
```

Comment:
This should not be needed.

Reply:
see https://github.com/cloudflare/workers-sdk/pull/6525#discussion_r1722039312

Reply:
my understanding from that comment is that we can remove this?